bad_code.java

```diff
@@ -8,14 +8,15 @@ public static void main(String[] args) {
 }
 
 class ParentClass {
-    public int someNumber = 10;
-    public String someString = "Hello";
+    public int number = 10;
+    public String string = "Hello";
 
     public void doSomething() {
         System.out.println("Doing something in ParentClass");
-        someNumber++;
-        someString += " Worlds";
-        System.out.println(someString);
+        number++;
+        System.out.println(number);
+        string += " Worlds";
```

Comment:
Use StringBuilder for string concatenation in loops to improve performance. [medium]